resources/views/pages-legacy/gameSearch.blade.php

```diff
@@ -41,11 +34,8 @@
     echo "<p>Show:</p>";
     echo "<select class='w-full sm:w-auto' onchange='window.location = \"/gameSearch.php?o=0&p=$method&i=\" + this.options[this.selectedIndex].value'>";
     foreach ($consoleList as $nextConsoleID => $nextConsoleName) {
-        // 0 is "All Consoles". Don't show consoles that haven't been rolled out yet.
-        if ($nextConsoleID == 0 || isValidConsoleId($nextConsoleID)) {
```

Comment:
This is handled by the `active` scope. The "0" value is added after executing the query.